src/sizzle.js

```diff
@@ -846,6 +846,12 @@ setDocument = Sizzle.setDocument = function( node ) {
 				rbuggyQSA.push( ":enabled", ":disabled" );
 			}
 
+			// Support: Firefox <=3.6 - 5 only
+			// Old Firefox doesn't throw on a badly-escaped identifier.
+			el.querySelectorAll( "x\\\f" );
```

Comment:
Is the `x` necessary?

Reply:
It might not be... I wasn't sure what should be the behavior with a trailing slash as tag names can't start with such characters, can they?

Reply:
U+000C FORM FEED is not allowed _anywhere_ in an element name, in either [HTML](https://html.spec.whatwg.org/multipage/parsing.html#tag-name-state) or [XML](https://www.w3.org/TR/2008/REC-xml-20081126/#sec-starttags). But that's irrelevant to CSS, in which we expect an error from any selector in which a backslash that should start an escape sequence is immediately followed by a CSS newline (including U+000C) but Firefox before version 6 does not reject.

Reply:
Ah, right; what's allowed in the host environment and what's allowed in CSS selectors are not the same things. I've updated the test.